frontend/src/metabase/admin/settings/containers/SettingsEditorApp.jsx

```diff
@@ -82,8 +83,12 @@ export default class SettingsEditorApp extends Component {
             }
 
             this.refs.layout.setSaved();
-            let val = (setting.key === "report-timezone" || setting.type === "boolean") ? setting.value : "success";
```

Comment:
I think we're no longer reporting the value of `report-timezone` in this refactor.